input/cql/COVID19EmergencyDeptSummary.cql

```diff
@@ -99,9 +110,19 @@ define TreatmentSummary:
   else
   {
     {
-      TreatmentType: 'NonPharmacologic',
+      TreatmentType: 'NonPharmaMildModerate',
+      PriorityOrder: 0,
+      Recommend: "Recommend MildAndModerate Non-Pharmacologic Treatment"
+    },
+    {
+      TreatmentType: 'NonPharmaSevere',
+      PriorityOrder: 0,
+      Recommend: "Recommend Severe Non-Pharmacologic Treatment"
+    },
+    {
+      TreatmentType: 'NonPharmaCritical',
       PriorityOrder: 0,
```

Comment:
May be there is a better way to handle this, we can create two wrapping methods to return `TreatmentType` and `Recommend`.